test/boost/commitlog_test.cc

```diff
@@ -205,6 +205,29 @@ SEASTAR_TEST_CASE(test_commitlog_new_segment){
     });
 }
 
+SEASTAR_TEST_CASE(test_commitlog_new_segment_512_align){
+    commitlog::config cfg;
+    cfg.commitlog_segment_size_in_mb = 1;
+    cfg.disk_write_alignment = 512;
```

Comment:
This may fail on a test system that requires larger alignment (e.g. sector size = 4096).

Reply:
True. Should probably change the code in segment alloc to use a max of config and segment alignment. Otoh, makes the config parameter rather pointless. So maybe just kill it?

Reply:
Killed it.